packages/forms/src/index.tsx

```diff
@@ -268,7 +273,7 @@ const TextAreaField = forwardRef<
     <textarea
       {...tagProps}
       id={props.id || props.name}
-      ref={(element) => {
+      {...(element: HTMLTextAreaElement | any) => {
```

Comment:
Using `any` was the only type that didn't give an error. I'm no TS expert so assuming there's a better way to handle this.

Note: multiple occurrences below as well.